pages/manual-content/index.html

```diff
@@ -1,6 +1,6 @@
 ---
 layout: main.njk
-title: California Coronavirus COVID-19 Response
+title: COVID19.CA.GOV
```

Comment:
Did you want this as the title?

Reply:
Will this new title affect SEO?

Main header: California Coronavirus COVID-19 Response
H1:  California Coronavirus COVID-19 Response

Title Current:  <title> Coronavirus COVID-19 Response Portal</title>

Notes: The title will show on google search results I would add State of California  to the main title.

Reply:
Seems like the GOV was ok with it and it doesn't need to be translated since it is a URL

Reply:
@abquirarte

Reply:
Mhm. I worry about SEO after this. Let's go for it but stay open minded.... cc @carterm for awareness.